### PR TITLE
fixed design changes on search screen

### PR DESCRIPTION
**Pull request by tjcarrel**

### CODE REVIEW

File: Cornell Sun/Cornell Sun/Search Section/SearchSectionController.swift [hunk 1]
```diff
@@ -58,22 +58,26 @@ class SearchSectionController: ListSectionController {
             // swiftlint:disable:next force_cast
             let cell = collectionContext!.dequeueReusableCell(of: CategoryCell.self, for: self, at: index) as! CategoryCell
             cell.post = entry
+            cell.layoutMargins = UIEdgeInsets.zero
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Search Section/SearchSectionController.swift]: can you change all of these to `.zero`?

File: Cornell Sun/Cornell Sun/SearchViewController.swift [hunk 2]
```diff
@@ -76,6 +76,8 @@ class SearchViewController: UIViewController, UITableViewDelegate {
         tableView.tableFooterView = UIView()
         tableView.delegate = self
         tableView.dataSource = self
+        tableView.layoutMargins = UIEdgeInsets.zero
+        tableView.separatorInset = UIEdgeInsets.zero
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/SearchViewController.swift]: same here, `.zero`

File: Cornell Sun/Cornell Sun/SearchViewController.swift [hunk 3]
```diff
@@ -107,6 +109,16 @@ class SearchViewController: UIViewController, UITableViewDelegate {
         searchController.searchBar.delegate = self
         searchController.definesPresentationContext = true
         searchController.dimsBackgroundDuringPresentation = false
+        searchController.searchBar.placeholder = "Search For Articles"
+        searchController.searchBar.setPositionAdjustment(UIOffset(horizontal: 10, vertical: 0), for: .search)
+        if let textField = searchController.searchBar.subviews.first?.subviews.compactMap({ $0 as? UITextField }).first {
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/SearchViewController.swift]: remove the parentheses around compactMap, i think you can just do `compactMap{ $0 as? UITextField }.first`

File: Cornell Sun/Cornell Sun/Search Section/SearchSectionController.swift (hunk 1 above)
Comment (AAAstorga) [reply Cornell Sun/Cornell Sun/Search Section/SearchSectionController.swift]: +1

File: Cornell Sun/Cornell Sun/SearchViewController.swift (hunk 3 above)
Comment (tjcarrel) [reply Cornell Sun/Cornell Sun/SearchViewController.swift]: Tried it, Xcode is requiring the parenthesis.

**Pull request merged**